CONTRIBUTING.md

```diff
@@ -6,14 +6,26 @@ Vielen Dank, dass Du bei OctoAwesome mitmachen möchtest und ein Teil davon werd
 
 Bitte beachte bei allem immer unseren [Code of Conduct](CODE_OF_CONDUCT.md), damit wir gemeinsam friedlich zusammenarbeiten können.
 
-<!-- ## FAQ's
-
-Häufig Gestellte fragen, findest Du auf dem GitHub Wiki oder im octoawesome wiki -->
-
 ## Coding
 
-Wenn Du mit programmieren möchtest, beachte bitte unsere Richtlinien zum [Codestyle](http://wiki.octoawesome.net/wiki/Coding_Styles) und unseren [Code of Conduct](CODE_OF_CONDUCT.md). Das heißt, wenn Du größere Änderungen im Code machst, produziere dazu ein Video, einen Stream oder einen Blog beitrag.
+Wenn Du mit programmieren möchtest, beachte bitte unsere Richtlinien zum [Codestyle](http://wiki.octoawesome.net/wiki/Coding_Styles) und unseren [Code of Conduct](CODE_OF_CONDUCT.md).
 
 ## Wiki
 
-Natürlich kann man nicht nur durch Programmieren mithelfen, sondern auch am [Wiki](http://wiki.octoawesome.net/wiki/Hauptseite) mitschreiben. Beachte hierzu bitte unsere [Wiki Richtlinien](http://wiki.octoawesome.net/wiki/Styleguide).
\ No newline at end of file
+Natürlich kann man nicht nur durch Programmieren mithelfen, sondern auch am [Wiki](http://wiki.octoawesome.net/wiki/Hauptseite) mitschreiben. Beachte hierzu bitte unsere [Wiki Richtlinien](http://wiki.octoawesome.net/wiki/Styleguide).
```

Comment:
```suggestion
Natürlich sind keine Programmierkenntnisse für die Mitarbeit notwending, sondern es kann auch am [Wiki](http://wiki.octoawesome.net/wiki/Hauptseite) mitgeschrieben werden. Beachte hierzu bitte unsere [Wiki Richtlinien](http://wiki.octoawesome.net/wiki/Styleguide).
```

Aber ergibt das noch so Sinn? Weil Styleguide wollen wir loswerden und beim Wiki eventuell den großen Teil sowieso neu machen?

Reply:
Styleguide würde ich tatsächlich einfach raus machen

Reply:
Habe es mal herausgenommen und dadurch ersetzt, dass man in den Issues mitdiskutieren darf

Reply:
Na aber der StyleGuide für das Wiki gilt doch noch

Reply:
Stimmt, nicht gesehen, dass das der Wiki Style ist. Aber wenn wir das Wiki ggf. auch neu machen, damit es hier mit auf Github ist, dann sollten wir da vermutlich auch noch mal drüber gehen